### ISSUE TITLE
Plan to add StyleGAN2 baseline for experiments

### ISSUE THREAD

**Issue by rangwani-harsh** [POSTECH-CVLab/PyTorch-StudioGAN#38]
Hi Minguk,

Any plans for implementing the StyleGAN2 generator architecture in the library? Are there some issues which are making it hard to implement in this framework?

Thanks for the great library.

**Comment by MINGUKKANG** [comment]
Hi! 

Thank you so much for your interest in my project.

Recently, many researchers use the generator architecture used in StyleGAN_V2 to verify their method. I also believe that implementing the generator architecture with PyTorch will be helpful for deep learning lovers. However, I am preparing a submission for ICCV and do not have enough time to implement and verify the model. 

After finishing the work, I will add the generator model of StyleGAN_v2.

Thank you.

Sincerely, 

Minguk

**Comment by MINGUKKANG** [comment]
Finally we add StyleGAN2 code!

Please refer to ReadME and [./src/configs/CIFAR10/StyleGAN2.yaml](https://github.com/POSTECH-CVLab/PyTorch-StudioGAN/blob/master/src/configs/CIFAR10/StyleGAN2.yaml) for more details.

**Comment by rangwani-harsh** [comment]
Thanks, @MINGUKKANG for your great work! We will keep this open till the performance (FID) gets verified.

**Comment by MINGUKKANG** [comment]
The synthesis results reported by (ReACGAN paper)[https://arxiv.org/abs/2111.01118] are from the current version of StudioGAN.

I will upload the logs, checkpoints, and configurations of the experiments soon.

Thank you.

**Comment by MINGUKKANG** [comment]
Thank you. 

We add configurations, checkpoints, and logs for StyleGAN2.

Please refer to the ReadME file for more details. 

Best, 

Minguk

**Comment by rangwani-harsh** [comment]
Thanks for your herculean effort!! The StudioGAN library is better than ever. Can we also obtain styleGAN evaluation metrics on the same validation set for CIFAR-10 as well?

**Comment by MINGUKKANG** [comment]
Definitely, you can. 

Using -ref "test" instead of -ref "train".

Thank you.
